packages/react-component-library/src/components/Timeline/__tests__/Timeline.test.tsx

```diff
@@ -2094,6 +2094,47 @@ describe('Timeline', () => {
     })
   })
 
+  describe('when `endDate` is before `startDate`', () => {
+    let consoleErrorSpy: jest.SpyInstance
+
+    beforeAll(() => {
+      timezoneMock.register('Europe/London')
+    })
+
+    beforeEach(() => {
+      consoleErrorSpy = jest.spyOn(global.console, 'error')
```

Comment:
Call `consoleErrorSpy.mockRestore()` in an `afterEach()`?

(Could turn on https://jestjs.io/docs/configuration#restoremocks-boolean, but looks like we'd need to fix some tests...)